VideoLocker/build.gradle

```diff
@@ -84,20 +84,28 @@ dependencies {
     }
     compile fileTree(dir: 'libs', include: '*.jar')
     compile 'com.facebook.android:facebook-android-sdk:3.22.+'
-    compile 'com.google.android.gms:play-services:6.5.+'
+    compile 'com.google.android.gms:play-services:7.5.0'
```

Comment:
Good point. I am not entirely sure, but i am guessing that we are only using the following:
`com.google.android.gms:play-services-plus`
`com.google.android.gms:play-services-identity`
`com.google.android.gms:play-services-analytics`
`com.google.android.gms:play-services-gcm`
`com.google.android.gms:play-services-location`
FYI @1zaman